aicsimageio/readers/default_reader.py

```diff
@@ -126,16 +128,12 @@ def _guess_dim_order(shape: Tuple[int]) -> str:
         return Reader._guess_dim_order(shape)
 
     @property
-    def scenes(self) -> Tuple[int]:
+    def scenes(self) -> Tuple[str]:
```

Comment:
just trying to remember, why Tuple and not list?

Reply:
I personally wanted this to be immutable. i.e. someone cant just `img.scenes.append("nice-scene-bro")` but there are numerous ways to get around this as well. Just preference I guess?